core/trino-main/src/main/java/io/trino/operator/scalar/MathFunctions.java

```diff
@@ -1398,4 +1386,30 @@ private static Double mapL2Norm(Block map)
 
         return Math.sqrt(norm);
     }
+
+    @Description("Black-Scholes equation for a Euro vanilla call option")
+    @ScalarFunction("black_scholes_euro_vanilla_call")
+    @SqlType(StandardTypes.DOUBLE)
+    public static double blackScholesEuroVanillaCall(@SqlType(StandardTypes.DOUBLE) double S, @SqlType(StandardTypes.DOUBLE) double K, @SqlType(StandardTypes.DOUBLE) double T, @SqlType(StandardTypes.DOUBLE) double r, @SqlType(StandardTypes.DOUBLE) double sigma) {
```

Comment:
Instead of naming argument as S, K, etc, can we have a proper expansion for this. It improves the code readability. Can we place each arguments in new line ? WDYT ?

Reply:
Renamed args and placed on newline. Please let me know if it looks okay @Praveen2112